eng/deploy.yaml

```diff
@@ -81,24 +124,24 @@ stages:
     - task: AzurePowerShell@3
       displayName: eng\firewall.ps1 -Add
       inputs:
-        azureSubscription: ${{ parameters.Subscription }}
-        ScriptPath: $(Pipeline.Workspace)/ReleaseUtilities/firewall.ps1
+        azureSubscription: $(Subscription)
```

Comment:
Sadly this will not work like this IIRC.
The azureSubscription parameter must be a constant during the execution of the pipeline, a template parameter is. This variable reference isn't.

Reply:
I tested the variables defined such in the context of pipeline triggers - https://dev.azure.com/dnceng/internal/_build/results?buildId=479558&view=results 

The variable values do get populated but the variable groups don't as I noticed in my test just now - https://dev.azure.com/dnceng/internal/_build/results?buildId=483500&view=results

Reply:
The variables were populated - https://dev.azure.com/dnceng/internal/_build/results?buildId=483603&view=results

Reply:
Moved to templates